integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -99,6 +99,11 @@ class ApplicationsIT extends ITSupport {
         }
     }
 
+    @Test
+    void testClientIsReady() {
+        assertThat client.isReady(), is(true)
```

Comment:
Avoid using `is(true)` or `is(false)` it does not provide a reason when it fails, instead set the reason manually:
```suggestion
        assertThat "Expected client to be ready", client.isReady()
```

Or something like that